core/src/main/java/org/apache/accumulo/core/clientImpl/NamespaceOperationsHelper.java

```diff
@@ -72,9 +72,7 @@ public void removeIterator(String namespace, String name, EnumSet<IteratorScope>
     if (!exists(namespace))
       throw new NamespaceNotFoundException(null, namespace, null);
     Map<String,String> copy = new TreeMap<>();
-    for (Entry<String,String> property : this.getProperties(namespace)) {
-      copy.put(property.getKey(), property.getValue());
-    }
+    this.getConfiguration(namespace).forEach(copy::put);
```

Comment:
This could be:

```java
Map<String,String> copy = Map.copyOf(this.getConfiguration(namespace));
```

(so long as we aren't making changes to the copy, because this would make an immutable copy)

There are a few such occurrences, if you want to change them.